app/components/MemberManagement.tsx

```diff
@@ -0,0 +1,168 @@
+import { Link } from "@remix-run/react";
+import { Angle_down, Arrow_right } from "~/components/ui/svgs";
+import { useState, ChangeEvent } from "react";
+import { Search, Check } from "lucide-react";
+
+function MemberManagement() {
+  const [search, setSearch] = useState<string>("");
+  const [filter, setFilter] = useState<string>("All");
+  const [display, setDisplay] = useState<string>("none");
+  const handleSearch = (e: ChangeEvent<HTMLInputElement>) => {
+    setSearch(e.target.value);
+  };
+
+  //   account type btn
+  const accountType_btn: string[] = [
+    "All",
+    "Members",
+    "Suspended",
+    "Left workspace",
+  ];
+
+  interface User {
```

Comment:
This bad code, please move these static data to another file and import it into this file.